src/components/clients/ClientForm.js

```diff
@@ -36,16 +67,43 @@ class ClientForm extends React.Component {
   }
 
   render() {
+    let authRedirectUrls = [];
+    for (let i = 0; i < this.state.authRedirectUrlsCount; i++) {
+      authRedirectUrls.push(
+        <input value={this.state.authRedirectUrls[i]} onFocus={this.onFocus}
+               onChange={this.onRedirectUrlChange} id={i} key={i} type="text"
+               name="authRedirectUrls" className="form-control"/>
```

Comment:
I don't think we use this class anywhere ?

Maybe the class could go on the (to be created ;) ) fieldset too ?